test/float/get_rm_invalid.c

```diff
@@ -0,0 +1,13 @@
+#include "smack.h"
+#include <fenv.h>
+
+// @expect verified
+
+int main(void) {
+  int rm = fegetround();
+  assume(rm < 1 || rm > 5);
```

Comment:
Hmmm, I don't get this regression. What is it testing for? Please clarify.

Reply:
This tests for the fact that `fegetround()` returns a negative value if the rounding mode cannot be determined. In other words, if rm was not assigned a value between 1 and 5 (which represent the range of possible rounding modes, then it must be negative.